backend/coins/coin/codes.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Shift Crypto AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package coin
+
+// Code represents a unique coin code. Usually the coin acronym in lowercase. ERC20-Tokens are also coins.
+type Code string
+
+const (
+	// CodeBTC is Bitcoin.
+	CodeBTC Code = "btc"
+	// CodeTBTC is Bitcoin Testnet.
+	CodeTBTC Code = "tbtc"
+	// CodeRBTC is Bitcoin Regtest.
+	CodeRBTC Code = "rbtc"
+	// CodeLTC is Litecoin.
+	CodeLTC Code = "ltc"
+	// CodeTLTC is Litecoin Testnet.
+	CodeTLTC Code = "tltc"
+	// CodeETH is Ethereum.
+	CodeETH Code = "eth"
+	// CodeTETH is Ethereum Ropsten.
+	CodeTETH Code = "teth"
+	// CodeRETH is Ethereum Rinkeby
+	CodeRETH Code = "reth"
+	// CodeERC20TEST is an arbitrarily picked test ERC20 token on Ropsten.
+	CodeERC20TEST Code = "erc20Test"
+	// If you add coins, don't forget to update `testnetCoins` below.
+	// There are some more coin codes for the supported erc20 tokens in erc20.go.
+)
+
+// TestnetCoins is the subset of all coins which are available in testnet mode.
+var TestnetCoins = map[Code]struct{}{
```

Comment:
on the other hand, there is only one use of it, and there might be an arbitrarily large number of erc20 tokens that would have to be repeated here. imho pass for now, ok?

Reply:
yeah, sure. sounds good. although even 100s of entries would be ok with a bool map because compared to the rest of memory usage it's a drop in the ocean.

Reply:
not worried about memory, more about maintaining multiple places with many repeated lines ;)